### DISCUSSION TITLE
SSH switch to running process

### DISCUSSION THREAD

**Discussion by julius1504** [jef/streetmerchant#1690] [Help] [answered]
Hello,
I have the bot set up on a VPS that I access via SSH. Everything seems to work, including Telegram notifications.
When I exit the SSH session and leave the bot running, e.g. when I restart my PC, how can I go back to the running bot process? I mean the page where it constantly checks the stock and either says out of stock or in stock.
Do I just do another _npm run start_ or does this start a second process (and a thrid, and a forth....)?
Is there a command to go back to the page that I see when I initially run _npm run start_?
Thanks guys

**Comment by wofnull** [comment, marked as answer]
you could run the npm run start command in a screen terminal on ssh so you are  able to reconnect to the screen session where the script is open running.

For more info:
https://linux.die.net/man/1/screen

**Reply by julius1504** [reply]
that did the trick, thank you!
